Runtime/Scripts/WebRTC.cs

```diff
@@ -909,14 +1089,35 @@ internal static class NativeMethods
         [DllImport(WebRTC.Lib)]
         [return: MarshalAs(UnmanagedType.LPStr)]
         public static extern string IceCandidateGetSdpMid(IntPtr candidate);
+#else
+        [DllImport(WebRTC.Lib)]
+        public static extern string IceCandidateGetCandidate(IntPtr candidate);
+        [DllImport(WebRTC.Lib)]
+        public static extern int IceCandidateGetSdpLineIndex(IntPtr candidate);
```

Comment:
Is this code needed?
It seems the same method above.

Also `IceCandidateGetSdp` and  `IceCandidateGetSdpMid` are same from original one.

Reply:
This seems to be the case indeed, so the code is not needed and can be put outside of the UNITY_WEBGL guard. Only `NativeMethods.IceCandidateGetCandidate` is different.

Reply:
I understood. Thanks.